src/sentry/services/hybrid_cloud/log/impl.py

```diff
@@ -87,6 +97,11 @@ def record_user_ip(self, *, event: UserIpEvent) -> None:
         outbox.save()
 
     def find_last_log(
-        self, *, organization_id: int | None, target_object_id: int | None, event: int | None
+        self,
+        *,
+        organization_id: int | None,
+        target_object_id: int | None,
+        event: int | None,
+        data: dict[str, str] | None,
```

Comment:
```suggestion
        data: dict[str, str] | None = None,
```

Reply:
With the additional default added, I haven't been able to figure out how to get the params to include the `data` field when calling `cls._get_abstract_rpc_methods()` [here](https://github.com/getsentry/sentry/blob/d77e47f6e86010c246aa352c629f96cc822cd55e/src/sentry/services/hybrid_cloud/rpc.py#L269C13-L269C23). Any ideas?

Reply:
You also need to change the signature of the abstract base class (LogService) to match.  When the signatures don't match it isn't included.